File: backend/src/main/java/br/ufpr/sept/so2/modules/solicitacoes/domain/SolicitacaoEvento.java
package br.ufpr.sept.so2.modules.solicitacoes.domain;

import java.time.OffsetDateTime;
import java.util.UUID;

public record SolicitacaoEvento(
        UUID id,
        String tipo,
        String estadoDe,
        String estadoPara,
        UUID atorId,
        String parecer,
        String payload,
        OffsetDateTime createdAt
) {
}

File: backend/src/main/kotlin/br/ufpr/sept/so2/modules/solicitacoes/domain/SolicitacaoEvento.kt
package br.ufpr.sept.so2.modules.solicitacoes.domain

import java.time.OffsetDateTime
import java.util.UUID

data class SolicitacaoEvento(
    val id: UUID,
    val tipo: String,
    val estadoDe: String?,
    val estadoPara: String,
    val atorId: UUID?,
    val parecer: String?,
    val payload: String?,
    val createdAt: OffsetDateTime,
)
